setup.cfg

```diff
@@ -2,9 +2,7 @@
 
 [coverage:run]
 # .coveragerc to control coverage.py and pytest-cov
-# Omit the test directory from test coverage
 omit =
-    #*/tests/*
```

Comment:
Why the change here?

Reply:
Looks like just some cleanup tossed in; was already a line with no effect.